core/templates/pages/creator-dashboard-page/creator-dashboard-page.component.spec.ts

```diff
@@ -400,20 +421,18 @@ describe('Creator dashboard controller', () => {
           title: '',
           status: 'private'
         })).toBe('Untitled');
+      });
 
-        ctrl.setExplorationsSortingOptions('numViews');
-        expect(ctrl.currentSortType).toBe('numViews');
-
-        expect(ctrl.sortByFunction({
-          numViews: '',
-          status: 'private'
-        })).toBe(0);
-
+      it('should not sort exploration list by rating when rating is not' +
```

Comment:
ditto

Reply:
When providing an empty object to sort the exploration list by ratings, the sort should not happen.